clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -189,8 +189,8 @@ func (p *profileMeasurement) gatherProfile(c clientset.Interface) ([]measurement
 
 		// Get the profile data over SSH.
 		// Start by checking that the provider allows us to do so.
-		if p.config.provider == "gke" || shouldGetAPIServerByK8sClient(p.config.componentName) {
-			// SSH to master is not possible in gke, but if the component is
+		if !isSSHSupported || shouldGetAPIServerByK8sClient(p.config.componentName) {
+			// SSH to master is not possible in gke/ or ks, but if the component is
```

Comment:
s/ks/eks/?

But, given that we have this isSSHSupported now, maybe let's remove the specific providers from the comments. Such comments will get quickly outdated once we disabled ssh for another provider. So maybe something like:
```
If we're here it means that ssh to master is not supported in the cluster. But if the component is kube-apiserver then we can get ...
```

WDYT?